### PR TITLE
infoschema/slow_query: fix shallow copy problem

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
[tidb-slow.log](https://github.com/pingcap/tidb/files/3250722/tidb-slow.log)
parse upper slow log and execute below:
```sql
root:information_schema> select user, db,digest from slow_query where user != "";
+----------+--------------------+------------------------------------------------------------------+
| user     | db                 | digest                                                           |
+----------+--------------------+------------------------------------------------------------------+
| riv,Crea |                    | ow_db_priv,Super_priv,Execute_priv,Create_view_priv,Show_view_pr |
| nternal: | 0381ff1b5c2994d657 | s_priv:pseudo                                                    |
|          |                    | # Num_cop_tasks: 1                                               |
|          |                    | # Cop_proc_avg: 0 Cop_proc_p90:                                  |
| lumns_pr |  Txn_start_ts: 408 | 00136655                                                         |
|          |                    | # Wait_time: 0.001 Request_count: 1 Total_keys: 1                |
|          |                    | # Is_                                                            |
+----------+--------------------+------------------------------------------------------------------+
3 rows in set
```
There are some wrong value in `user`, `db`.

after fix:

```sql
root:information_schema> select user, db,digest from slow_query where user != "";
+----------+--------------------+------------------------------------------------------------------+
| user     | db                 | digest                                                           |
+----------+--------------------+------------------------------------------------------------------+
| root@::1 |                    | f106b0f6c630f454d31b008fbb034fda10f2441fd0477a9990e22b7df313f59a |
| root@::1 | information_schema | b4dae6a771c1d84157dcc302bef38cbff77a7a8ff89ee38302ac3324485454a3 |
| root@::1 | information_schema | b4dae6a771c1d84157dcc302bef38cbff77a7a8ff89ee38302ac3324485454a3 |
+----------+--------------------+------------------------------------------------------------------+
3 rows in set
```

### What is changed and how it works?
Fix this by add copy.

### Check List 

Tests 

 - I can't reproduce this in a simple unit test ...

### CODE REVIEW

File: infoschema/slow_log.go [hunk 1]
```diff
@@ -188,6 +188,7 @@ type slowQueryTuple struct {
 }
 
 func (st *slowQueryTuple) setFieldValue(tz *time.Location, field, value string) error {
+	value = string(hack.String([]byte(value)))
```
Comment (zz-jason) [inline infoschema/slow_log.go]: how about using `stringutil.Copy()`?

Comment (crazycs520) [reply infoschema/slow_log.go]: Great, Thanks.

**Comment by coocood** [comment]
LGTM

**Pull request merged**